src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -54,6 +55,21 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+    /**
+     * Parses a {@code String goodName} into a {@code goodName}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code goodName} is invalid.
+     */
+    public static GoodName parseGoodName(String goodName) throws ParseException {
+        requireNonNull(goodName);
+        String trimmedName = goodName.trim();
+        if (!GoodName.isValidGoodName(trimmedName)) {
+            throw new ParseException(GoodName.MESSAGE_CONSTRAINTS);
+        }
+        return new GoodName(trimmedName);
+    }
+
```

Comment:
Sorry, please change my parseGood to parseGoodName instead, I forgot to rename it

Reply:
Changed and removed the duplicate method